cookienotice/templates/cookienotice.tpl

```diff
@@ -0,0 +1,23 @@
+<style type="text/css">
+	#cookienotice-box {
```

Comment:
Please move the CSS in the `cookienotice.css` file.

Reply:
done